File: codes/fibo/fibo.ml

open Sys;;

let rec fibo i = if i <=1 then i else fibo(i-1)+fibo(i-2);;

let valeur =  fibo (int_of_string(argv.(1)));;
let strres =  "fibo "^argv.(1)^": "^(string_of_int valeur) ;;
print_endline strres;;

File: codes/fibo/fibo.java
public class fibo
{


	static long n = 20;
	static long nmin = 10;
	static long res;


	static  long fibo(long n)
	{
		if (n <= 1) 
			return n;
		return fibo(n-1)+fibo(n-2);
	}
	
	public static void  main(String[] argv)
	{
		n = new Long(argv[0]);
		long t1=0,t2=0;
		double times = 20;
		int i;
		System.out.printf("fibo %d: %d\n",n,fibo(n));
	}


}



